test/integration/lets_encrypt_cert/lets_encrypt_test.rb

```diff
@@ -13,13 +30,25 @@
   its(:content) { should match 'BEGIN CERTIFICATE' }
 end
 
-describe file(path_to_lets_encrypt_key) do
-  it { should exist }
-  it { should be_file }
-  it { should be_mode 0o640 }
-  it { should be_owned_by 'root' }
-  it { should be_grouped_into key_group(node) }
-  its(:content) { should match 'BEGIN RSA PRIVATE KEY' }
+describe x509_certificate(path_to_self_signed_cert(node)) do
+  its('version') { should eq 2 }
+  its('key_length') { should eq 2048 }
+  its('signature_algorithm') { should eq 'sha256WithRSAEncryption' }
+  its('validity_in_days') { should be > 89 }
+  its('validity_in_days') { should be < 91 }
+
+  its('extensions') { should include 'subjectAltName' }
+
+  # Need to update these
```

Comment:
Should these be updated? Looks like the recipe actually sets these now?

Maybe even removed since the real ones are tested below?